src/elements/content-uploader/__tests__/UploadsManagerAction.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import UploadsManagerAction from '../UploadsManagerAction';
+
+describe('elements/content-uploader/UploadsManagerAction', () => {
+    test('should render correctly with hasMultipleFailedUploads as true', () => {
+        render(<UploadsManagerAction hasMultipleFailedUploads={true} onClick={jest.fn()} />);
+
+        expect(screen.getByText('Resume All')).toBeInTheDocument();
```

Comment:
should we do the same thing we did below where we asserted one and verifying the other is not there?

Reply:
The assertion check we did below is not applicable for this case since `Resume All` includes substring `Resume`